be/src/util/misc.h

```diff
@@ -22,4 +24,21 @@ namespace starrocks {
 // take a sleep with small intervals until time out by `sleep_secs` or the `stop_condition()` is true
 void nap_sleep(int32_t sleep_secs, const std::function<bool()>& stop_condition);
 
+#if defined(__GNUC__) || defined(__clang__)
+#define NOT_SUPPORT()                                                                        \
+    do {                                                                                     \
+        throw std::runtime_error(std::string("Not support method '") + __PRETTY_FUNCTION__ + \
```

Comment:
`runtime_error` will be printed by the stack_util as always, is it expected. `RuntimeException` is introduced in https://github.com/starRocks/StarRocks/pull/62098 , worth a double check.